scripts/src/main/resources/scripts/command/aws

```diff
@@ -29,43 +43,46 @@ function doPackageInstall()
   then
     doEcho "Installing AWS for Windows..."
     windows_path_to_package=$(cygpath -w "${path_to_package}")
-    powershell.exe -Command "Start-Process msiexec.exe -verb runas -Wait -ArgumentList '/I ${windows_path_to_package}\aws-${current_version}-AWSCLIV2.msi /quiet'"
+    powershell.exe -Command "Start-Process msiexec.exe -verb runas -Wait -ArgumentList '/I ${windows_path_to_package}\aws-${AWS_VERSION}-windows.msi /quiet'"
```

Comment:
function is not really reusable as `${AWS_VERSION}` has to be initialized what happens inside `doSetup`.
Unfortunately bash has no concept of visibilities like `private` vs. `public`.
However, I understood that you extracted this function to increase readability (`doSetup` gets shorter and this "package installation" is grouped in a function with a self explanatory name. So all fine...

Reply:
Now I add the version as the second parameter. The better way.

Reply:
nice solution. Thanks.